app/Appointment.php

```diff
@@ -39,4 +39,10 @@ public function treatment(): BelongsTo
         return $this->belongsTo(Treatment::class);
     }
 
+    public function getTreatmentEndTimeAttribute()
```

Comment:
Can we bind the return on this?
```suggestion
    public function getTreatmentEndTimeAttribute(): string
```